src/adaptors/articles.ts

```diff
@@ -0,0 +1,13 @@
+import { Iri, JsonLdObj } from 'jsonld/jsonld-spec';
+
+export interface Articles extends Iterable<JsonLdObj> {
```

Comment:
Adding default here introduces the lint error: 

```
/app/src/adaptors/articles.ts
  13:2  error  Missing semicolon  semi
```

Reply:
```typescript
interface Articles {}

export default Articles;
```

works, but then runs into https://github.com/typescript-eslint/typescript-eslint/issues/342. Let's disable `no-undef` (in a separate PR).

Reply:
In case it helps, default exports do have downsides:
https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

Reply:
Hmm, eslint will currently complain if trying to do this on a class. Favour sticking with defaults for now (for consistency), but one to have a think about.